processing/data_collection/gazette/spiders/mg_itauna.py

```diff
@@ -0,0 +1,36 @@
+import datetime as dt
+
+from dateparser import parse
+from scrapy import Request
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class MgItaunaSpider(BaseGazetteSpider):
+    TERRITORY_ID = "3133808"
+    name = "mg_itauna"
+    allowed_domains = ["itauna.mg.gov.br"]
+    start_urls = ["https://www.itauna.mg.gov.br/portal/diario-oficial/"]
+
+    def parse(self, response):
+        diarios = response.xpath("//div[@class='d_e_modelo_diario']")
+        for diario in diarios:
+            url = (
+                "https://www.itauna.mg.gov.br"
+                + diario.css("a")[1].extract().split('"')[1]
+            )
+            date = diario.xpath("div/span/text()").extract()[0].split()[0]
```

Comment:
Newer versions of Scrapy are favoring `.getall()` as a replacement for `.extract()`.

Reply:
Also, `.get()` is a nice shorcut for `.getall()[0]` :)

Reply:
Maybe using `.re_first("\d{2}/\d{2}/\d{4}")` could avoid potential problems in the future if they decide to change the order of the fields?